extensions/wikidata/tests/src/org/openrefine/wikidata/schema/WbReferenceExprTest.java

```diff
@@ -77,4 +79,10 @@ public void testSerialize()
             throws JsonProcessingException {
         JacksonSerializationTest.canonicalSerialization(WbReferenceExpr.class, expr, jsonRepresentation);
     }
+
+    @Test
+    public void testUnmodifiableList() {
+        List<WbSnakExpr> snakExprs = expr.getSnaks();
+        Assert.assertTrue(snakExprs.getClass().getName().contains("Unmodifiable"));
```

Comment:
This way of testing immutability might be a bit brittle - the tests could fail if we use other implementations of unmodifiable lists which have a different name (typically, the immutable variant you first used). Perhaps a simpler way to test that would be to try to modify the list and check that it throws an exception?
For instance

```
@Test(expectedExceptions = UnsupportedOperationException.class)
public void testUnmodifiableList() {
        expr.getSnaks().clear();
}
```